tests/components/FilterSearch.stories.tsx

```diff
@@ -78,25 +78,6 @@ DropdownSectioned.play = ({ canvasElement }) => {
   userEvent.type(canvas.getByRole('textbox'), 'name');
 };
 
-export const DropdownHighlight = (args: FilterSearchProps) => {
```

Comment:
We don't want this snapshot anymore?

Reply:
Yeah, since the first option is auto-highlighted the snapshot is the same as the normal dropdown story

Reply:
sgtm! I was thinking I wouldn't want to lose coverage on the {Tab} interactions but we have unit tests for it